picocrt/machine/arm/crt0.c

```diff
@@ -107,16 +108,80 @@ _cstart(void)
 
 extern char __stack[];
 
+#ifdef _PICOCRT_ENABLE_MMU
+
+#if __ARM_ARCH >= 7 && __ARM_ARCH_PROFILE != 'R'
+
+/*
+ * We need 2048 1MB mappings to cover the usual Normal memory space,
+ * which runs from 0x00000000 to 0x7fffffff
+ */
+#define MMU_MAPPING_COUNT 2048
```

Comment:
Is it actually safe only initializing half the page table? Doesn't that mean that any 0x8.... address will be translated to whatever happens to come after this page table?